### ISSUE TITLE
latest beta crashed several times, froze the camera, etc.

### ISSUE THREAD

**Issue by grinapo** [mapillary/mapillary_issues#2731]
Sent the debug logs to support@.

Apart from the camera API problem (different issue) this version have kept crashing, a few times got into uninterruptible wait, and there was a point where the cameta was not accessible through the API, and another I believe where the imagery was black. There seem to be several half-saved (half-black vertically) images in the upoaded sequence.
Also the program seem to haev forgotten that I haev set _distance_ based imagery _several times_, and been set back to time based. 
Also the focus want back to automatic from infinity (which is great to shoot images of the dust particles on the windshield! ;-)).
And lost GPS fix several times but it may or may not have been the fault of mapillary.
Possibly all of these are visible in the logs. (suject contain issue #2665)

**Comment by ovivoda** [comment]
@grinapo Please try the latest beta version on GooglePlay and let me know if it works better

**Comment by grinapo** [comment]
Sir, yes SIR. ;-)
(Have you fixed again the camera problem on S7? It's been goo for a while by setting old api, right now it's wrong regardless of the setting.)

**Comment by ovivoda** [comment]
@grinapo Oversharpened images on S7?

**Comment by grinapo** [comment]
@ovivoda Yes. I'll check the latest beta but I don't thing you've changed it since a few releases, and last one was bad. Shall we move this dicussion over there?

And for the actual error: I haven't used it recently, the weather was ugly all the time.

**Comment by jfburke3** [comment]
I want to reference #2598 here.  @grinapo does this look similar to your problem?

**Comment by grinapo** [comment]
Possibly related, also with CamAPI2. 

I am forcing API to legacy and have no problem, partially due to the intermittent oversharpening problem. Maybe I should see how Samsung handles user bugreports... :-S

**Comment by ovivoda** [comment]
@grinapo switching to old camera API did helped fix this?

**Comment by grinapo** [comment]
@ovivoda yes, again, forcing the old API seem to work and it's my permanent solution now. Last time I have switched back to API2 it worked for a few cases then it's been fucked himself again in a short while, and it's pretty annoying to have a long, interesting mapillary session without the imagery. ;-)

I have tried to contact Samsung but it's like an endless labyrinth, an infinite redirection to various parts of their communities without anyone having a clue. I should try their "developer community" but got delayed by summer holiday, thanks $LocalDeity. I'll try further, as time allows. 

(My S7 has been updated to android 8.0.0 recently so I should probably try again. Okay, okay, you're right, I will. :-D)
